src/polar/Pie.tsx

```diff
@@ -58,7 +58,7 @@ type PieLabelLine =
   | ((props: any) => ReactElement<SVGElement>)
   | SVGProps<SVGPathElement>
   | boolean;
-export type PieLabel<P = any> =
+export type PieLabel<P extends PieSectorDataItem = PieSectorDataItem> =
```

Comment:
does `PieSectorDataItem` contain the full set of properties that are possibly provided in `props`?

Reply:
I've come up with a more accurate type for the PieLabel props - indeed I was missing a couple of props. 

I've attempted to reuse some already-defined types to build the `labelprops`, such as `GeometrySector`, but I couldn't find perfect matches - there are often some extra props defined, albeit optional, but defined nonetheless. So bare with me as this is an imperfect solution.

Below, I've separated the properties defined in `PieSectorDataItem` to their own type that I named `PieSectorData` in order to avoid looping in `SectorProps`, which contains some items Not available in labelprops.

I added the missing `tooltipPayload` prop. tooltipPayload is defined as `any` because I could not find a type definition that fits.

Finally, I've added the type `PieLabelRenderProps` which will help avoid a breaking change, since its type definition contains `[key: string]: any` by default. 

This is an imperfect solution, but would improve dev experience until a more perfect solution can be worked on. I understand that its a huge amount of work.

```ts
interface PieSectorData {
  percent?: number;
  name?: string | number;
  midAngle?: number;
  middleRadius?: number;
  tooltipPosition?: Coordinate;
  value?: number;
  paddingAngle?: number;
  dataKey?: string;
  payload?: any[];
}

type PieLabelProps = PieSectorData &
  GeometrySector & {
    tooltipPayload?: any;
  } & PieLabelRenderProps;
  
export type PieSectorDataItem = PieSectorData & SectorProps
...
...
export type PieLabel<P extends PieLabelProps = PieLabelProps>
```

Reply:
> I've attempted to reuse some already-defined types

I am not worried about reusing types and type definitions. All of those are compiled away anyway so it doesn't increase the size of javascript that browser has to download.

The types are for other developers mostly - so it's more important that it documents the intention, than reuse.

Reply:
+1 Also not worried about re-use. @alexshenker can you double check once more that there haven't been any extra types added here? Once that is confirmed this looks fine